test_agent/python/testagent.py

```diff
@@ -288,29 +299,43 @@ def handle_uri_validate_command(json_msg):
             ACTION_COMMANDS.VALIDATE_URI,
             received_test_id=json_msg["test_id"],
         )
+    else:
+        send_to_test_manager(
+            {
+                "result": "False",
+                "message": "Nonexistent UriValidator function",
+            },
+            CONSTANTS.VALIDATE_URI,
```

Comment:
I don't see VALIDATE_URI in constants.py, maybe you meant ACTION_COMMANDS.VALIDATE_URI

Reply:
Great find!